www/_template/reference/configuration.md

```diff
@@ -124,235 +297,157 @@ module.exports = {
 
-#### devOptions.secure | `boolean`
 
-- Toggles whether or not Snowpack dev server should use HTTPS with HTTP2 enabled.
+### installOptions.packageLookupFields
 
-#### devOptions.out | `string` | Default: `"build"`
+**Type**: `string[]`
 
-- _NOTE:_ Deprecated, see `buildOptions.out`.
-- The local directory that we output your final build to.
+Set custom lookup fields for dependency `package.json` file entrypoints, in addition to the defaults like "module", "main", etc.
 
-### config.buildOptions
+Example: `"packageLookupFields": ["svelte"]`
 
-#### buildOptions.out | `string` | Default: `"build"`
+## config.buildOptions
 
-- The local directory that we output your final build to.
+**Type**: `object` (option name: value)
 
-#### buildOptions.baseUrl | `string` | Default: `/`
+Configure your final build.
 
-- In your HTML, replace all instances of `%PUBLIC_URL%` with this (inspired by the same [Create React App](https://create-react-app.dev/docs/using-the-public-folder/) concept). This is useful if your app will be deployed to a subdirectory. _Note: if you have `homepage` in your `package.json`, Snowpack will actually pick up on that, too._
+### buildOptions.out
 
-#### buildOptions.clean | `boolean` | Default: `true`
+**Type**: `string`
+**Default**: `"build"`
 
-- Set to `false` to prevent Snowpack from deleting the build output folder (`buildOptions.out`) between builds.
+The local directory that we output your final build to.
 
-#### buildOptions.metaDir | `string` | Default: `__snowpack__`
+### buildOptions.baseUrl
 
-- By default, Snowpack outputs Snowpack-related metadata such as [HMR](/concepts/hot-module-replacement) and [ENV](/reference/configuration#environment-variables) info to a folder called `__snowpack__`. You can rename that folder with this option (e.g.: `metaDir: 'static/snowpack'`).
+**Type**: `string`
 
-#### buildOptions.sourceMaps | `boolean` | Default: `false`
+**Default**: `/`
 
-- **_Experimental:_** Set to `true` to enable source maps
+In your HTML, replace all instances of `%PUBLIC_URL%` with this
 
-#### buildOptions.webModulesUrl | `string` | Default: `web_modules`
+Inspired by the same [Create React App](https://create-react-app.dev/docs/using-the-public-folder/) concept. This is useful if your app will be deployed to a subdirectory.
 
-- Rename your web modules directory.
+_Note: if you have `homepage` in your `package.json`, Snowpack will actually pick up on that, too._
 
-#### buildOptions.jsxFactory | `string` | Default: `React.createElement` (or `h` if Preact import is detected)
+### buildOptions.webModulesUrl
 
-- Set the name of the used function to create JSX elements.
+**Type**: `string`
 
-#### buildOptions.jsxFragment | `string` | Default: `React.Fragment` (or `Fragment` if Preact import is detected)
+**Default**: `web_modules`
 
-- Set the name of the used function to create JSX fragments.
+Rename your web modules directory.
```

Comment:
can you describe this a bit more: what's a web modules directory? rename it how? happy to help you on this one if it's confusing

Reply:
What about `The `web_modules` directory is where Snowpack builds your `node_modules` to after converting to ESM.`?

Reply:
lets chat over this in person when we meet today, it's a bit more subtle and I want to explain it in person.